### ISSUE TITLE
error  'error' is defined but never used 

### ISSUE THREAD

**Issue by mbrevda** [sweepline/eslint-plugin-unused-imports#105]
When using Eslint 9, I'm getting `error  'error' is defined but never used ` for simple try/catch blocks where the error variable isn't handled. Seems similar to this issue [here](https://github.com/sweepline/eslint-plugin-unused-imports/issues/100).

e.g.

```js
try {
  doSomething()
} catch (e) {
  /* continue anyway */
}

```

**Comment by nfabredev** [comment]
Following this [issue](https://github.com/sweepline/eslint-plugin-unused-imports/issues/93) and looking at the [docs](https://eslint.org/docs/latest/rules/no-unused-vars#caughterrorsignorepattern) I ended up with this configuration:

```
'unused-imports/no-unused-vars': [
        'error',
        {
          caughtErrors: 'all',
          caughtErrorsIgnorePattern: '^_',
          vars: 'all',
          varsIgnorePattern: '^_',
          args: 'after-used',
          argsIgnorePattern: '^_',
        },
      ],
```

**Comment by mbrevda** [comment]
Not sure I follow: what is the connection between catch blocks and imports?

**Comment by ic-768** [comment]
I have the same issue

**Comment by antfu** [comment]
This is working as expected. The error is reported with `unused-imports/no-unused-vars` not `unused-imports/no-unused-imports`.

**Comment by mbrevda** [comment]
oh thanks - I missed that!
